docs/integrations/mirroring.md

```diff
@@ -0,0 +1,47 @@
+---
+id: mirroring_integration
+title: Mirroring Integration
+---
+
+Server version 6.0.0 adds support for Mirroring Integrations. These integrations allow you to mirror incidents/tickets/cases from other services for example: Another XSOAR server, Cortex XDR, ServiceNow
+
+An example Mirroring Integration can be seen [here](https://github.com/demisto/content/tree/master/Packs/XSOARMirroring/Integrations/XSOARMirroring).
+
+Mirroring Integrations are developed the same as other Integrations. They provide a few extra configuration parameters and APIs.
+
+
+## Supported Server Version
+Mirroring Integration's YAML file _must_ have the following field `fromversion: 6.0.0`. This is because Mirroring Integrations are only supported from Server version 6.0.0 and onwards.
+
+
+## Required Parameters
+Mirroring integration should have the following parameters in the integration YAML file:
```

Comment:
These params need to be under `script` in the yml right? like `isfetch` right now?